src/BedethequeScraper2.py

```diff
@@ -3266,42 +3018,53 @@ def button_Click(self, sender, e):
             aWord = Translate()
             log_BD(TIMEOUTS,"",1)
 
-        if sender.Name.CompareTo(self._ButtonReset.Name) == 0:
-            for CBControl in self._tabPage2.Controls:
-                try:
-                    if CBControl.CheckState == CheckState.Checked:
-                        CBControl.CheckState = CheckState.Unchecked
-                    else:
-                        CBControl.CheckState = CheckState.Checked
-                except:
-                    pass
-            self._TBTags.Text = ""
+        elif sender.Name.CompareTo(self._ButtonCheckNone.Name) == 0:
+            for index in range(self._ScrapedDataCheckedListBox.Items.Count):
+                self._ScrapedDataCheckedListBox.SetItemCheckState(index, CheckState.Unchecked)
+                if index == self._scrapedData.keys().index('Web'):

```

Comment:
I don't get this part, why is Web Special? Shouldn't it be set by the line above anyway. You are using the same function to set it anyway.